src/__tests__/logger.test.js

```diff
@@ -1,16 +1,17 @@
 import config from '../__mocks__/config';
 import {
+    addStream,
+    createLogger,
     transportConsole,
     transportLoggly,
     skip,
     omit,
 } from '../logger';
 
 describe('create a new logger and transports', () => {
-    it('should return a winston logger instance', () => {
-        // xxx good oprotunity to try writing more graph oriented tests
-        // const logger = createLogger(config);
-        // expect(logger.level).toBe('info');
+    it.skip('should return a winston logger instance', () => {
```

Comment:
You're going to revise these later?

Reply:
Yes. Want to re-write to use nodule. But at this point guess I should wait for you to settle the `nodule api` from https://github.com/globality-corp/nodule-config/pull/8. What you propose is a little different from the current api.